crates/oxc_diagnostics/src/service.rs

```diff
@@ -128,20 +129,38 @@ impl DiagnosticService {
     /// Wrap [diagnostics] with the source code and path, converting them into [Error]s.
     ///
     /// [diagnostics]: OxcDiagnostic
-    pub fn wrap_diagnostics<P: AsRef<Path>>(
+    pub fn wrap_diagnostics<C: AsRef<Path>, P: AsRef<Path>>(
+        cwd: C,
         path: P,
         source_text: &str,
         source_start: u32,
         diagnostics: Vec<OxcDiagnostic>,
-    ) -> (PathBuf, Vec<Error>) {
-        let path = path.as_ref();
-        let path_display = path.to_string_lossy();
+    ) -> Vec<Error> {
+        #[cfg(test)]
+        let is_jetbrains_terminal = false;
+        #[cfg(not(test))]
+        let is_jetbrains_terminal =
+            std::env::var("TERMINAL_EMULATOR").is_ok_and(|x| x.eq("JetBrains-JediTerm"));
```

Comment:
https://github.com/biomejs/biome/blob/adaa65ce06b7882accd60a97207ad4403bde23cf/crates/biome_diagnostics/src/display.rs#L684

do we need to check `TERM_PROGRAM` as well?